src/Stratis.Bitcoin.Features.SignalR/IClientEventBroadcaster.cs

```diff
@@ -0,0 +1,7 @@
+namespace Stratis.Bitcoin.Features.SignalR
+{
+    public interface IClientEventBroadcaster
+    {
+        void Initialise();
```

Comment:
For consistency, we have it here as `Initialise` but in `IEventsSubscriptionService` as `Init`

Reply:
Done